apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktOppgaveService.kt

```diff
@@ -80,7 +82,11 @@ class AktivitetspliktOppgaveService(
     ): AktivitetspliktInformasjonBrevdata? {
         val oppgave = oppgaveService.hentOppgave(oppgaveId)
         val sak = sakService.finnSak(oppgave.sakId) ?: throw GenerellIkkeFunnetException()
-        aktivitetspliktBrevDao.lagreBrevdata(data.toDaoObjektBrevutfall(oppgaveId, sakid = sak.id))
+        val saksbehandler =
+            Kontekst.get().brukerTokenInfo
+                ?: throw IngenSaksbehandler("Fant ingen saksbehandler til lagring av brevdata for oppgave $oppgaveId")
+        val kilde = Grunnlagsopplysning.Saksbehandler.create(saksbehandler.ident())
+        aktivitetspliktBrevDao.lagreBrevdata(data.toDaoObjektBrevutfall(oppgaveId, sakid = sak.id, kilde = kilde))
```

Comment:
Kunne du ikke bare brukt `saksbehandler: String`? Vil da aldri være noe annet enn en Z-ident eller `EY` så vidt jeg vet

Reply:
Antok at tidspunkt for opprettet/endret var en del av tabellen, men ser nå at det ikke er det. Oh well